docs/src/intervention.html.jinja2

```diff
@@ -557,6 +557,34 @@
                         </div>
                     </div>
 
+                    <div class="row">
+                        <div class="col-md-3">
+                            <label>Discharge</label>
+                            <div class="form-group">
+                                <label for="rbrasdt">Date</label>
+                                {{ f.datepicker("rbrdd", "past") }}
+                            </div>
+                        </div>
+                        <div class="col-md-3">
+                            <label>Readmit within 30 days of discharge?</label>
+                            <div class="form-group">
+                                {{ f.inlineRadio("rbrthd", "rbrthd-n", "n", "No", true) }}
```

Comment:
There's actually a control for this N-Y Radio pattern. Use `f.inlineRadioNY(rbrthd, true)` in all of these sections where you're doing this.

Reply:
I'll verify. Thank you.

Reply:
I've replaced all of the cases where `inlineRadioNY` should be used, including all the ones I didn't add. It should be consistent throughout the intervention form.